recipes/feets/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "feets" %}
+{% set version = "0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "f74e539a385a182e9bddcc444c96639dd953d9cd04bd6865458817285e1fb207"
+
+build:
+  number: 0
```

Can this be `noarch`?
```suggestion
  number: 0
  noarch: python
```

---

yes can be noarch

---

Please, add ``noarch`` then!  :)